tests/integration/test_epicor.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
```

Comment:
I feel like these tests aren't enough; key logic of this connector is not tested here. We should have a test for each of the 4 parsing functions. They should be easy to write, though - an LLM can probably autogenerate some test XML.